sub-packages/bionemo-scdl/examples/example_notebook.ipynb

```diff
@@ -176,13 +175,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 11,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'output_temp_dir' is not defined",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
+      "Cell \u001b[0;32mIn[11], line 2\u001b[0m\n\u001b[1;32m      1\u001b[0m dataset_temp_dir\u001b[38;5;241m.\u001b[39mcleanup()\n\u001b[0;32m----> 2\u001b[0m \u001b[43moutput_temp_dir\u001b[49m\u001b[38;5;241m.\u001b[39mcleanup()\n",
+      "\u001b[0;31mNameError\u001b[0m: name 'output_temp_dir' is not defined"
+     ]
+    }
```

Comment:
I think you meant to re-run the notebook after the fix

Reply:
done